src/applications/disability-benefits/all-claims/tests/pages/terminallyIll.unit.spec.jsx

```diff
@@ -23,16 +25,20 @@ describe('Terminally Ill', () => {
       />,
     );
 
-    // Expect one question with two radio inputs
-    expect(form.find('.form-radio-buttons').length).to.equal(1);
-    expect(form.find('.form-radio-buttons input').length).to.equal(2);
-    expect(form.find('input').length).to.equal(2);
-    form.unmount();
+    getByText('High Priority claims');
+
+    const question = container.querySelector('va-radio');
+    expect(question).to.have.attribute('label', 'Are you terminally ill?');
+
+    expect(container.querySelector('va-radio-option[label="Yes"', container)).to
+      .exist;
+    expect(container.querySelector('va-radio-option[label="No"', container)).to
+      .exist;
```

Comment:
Really like how on this "it should render" test we're checking for the presence of 1. the question (label) and 2. each radio button and their respective label. Is this something we could do for each of these pages?